Makefile

```diff
@@ -12,24 +12,27 @@ format:
 mypy:
 	poetry run mypy cashu --ignore-missing
 
+flake8:
+	poetry run flake8
```

Comment:
Can you also add `flake8` to the `format` check just above on L7?

(Sidenote, Makefile calling `make` as `format:` currently does is a bit weird, AFAIK you can just change it to this:

```makefile
format: isort black flake8 mypy
```

and it'll run all of those targets)

Reply:
cleaned it up a little, i would seperate the actual formatting with the checking. i added `make check` and those are running mypy, flake8, isort-check and black-check